config/__init__.py

```diff
@@ -17,7 +17,7 @@ class Config:
         'SLACK_API_TOKEN': 'slack_api_token',
         'SLACK_NOTIFICATION_CHANNEL': 'slack_notification_channel',
         'SLACK_ANNOUNCEMENT_CHANNEL': 'slack_announcement_channel',
-
+        'SLACK_PAIRING_CHANNEL': 'slack_pairing_channel',

```

Comment:
Can we move this to `OPTIONALS` and make this entire feature feature-toggled on this variable?